Samples/NetworkMediaStreamer/RTSPServer/inc/RTSPServer.h

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#pragma once
+#define DBGLEVEL 1
+
+class RTSPServer  : public winrt::implements<RTSPServer, IRTSPServerControl>
+{
+public:
+    RTSPServer(ABI::RTSPSuffixSinkMap *streamers, uint16_t socketPort, IRTSPAuthProvider *pAuthProvider, PCCERT_CONTEXT* serverCerts, size_t uCertCount)

```

Comment:
noexcept of catch the error?

Reply:
CreateRTSPServer is the caller to this and is catching exceptions here.

---
In reply to: [491104453](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491104453) [](ancestors = 491104453)